cmd/notify-elasticsearch.go

```diff
@@ -17,14 +17,20 @@
 package cmd
 
 import (
-	"encoding/hex"
-	"errors"
+	"context"
 	"fmt"
 	"io/ioutil"
+	"time"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/minio/sha256-simd"
-	"gopkg.in/olivere/elastic.v3"
+	"gopkg.in/olivere/elastic.v5"
+)
+
+var (
+	makeESError = makeNotificationErrorMaker("Elasticsearch Notifier Error")
+
+	esNFormatError = makeESError(`"format" value is invalid - it must be one of "%s" or "%s".`, formatNamespace, formatAccess)
```

Comment:
the format for error strings is . `err` is started at the beginning - for example look at `typed-errors.go` 

```
errESFormat
errESIndex
```